meson.build

```diff
@@ -747,6 +747,11 @@ if features['uchardet']
     dependencies += uchardet
 endif
 
+features += {'local-uuid': libavutil.version().version_compare('< 57.27.100')}
```

Comment:
I'd call this `lavu-uuid` or such and invert the value

Reply:
Why does this need a config flag? Can't you just use a preprocessor macro to check the lavu version inside the code?

Reply:
To avoid compiling it in when not needed.

Reply:
> I'd call this `lavu-uuid` or such and invert the value

I started off with this and then flipped it. Happy to switch it back.

Reply:
I thought the "this flag exists when the feature is missing" was weird and suggested this :smiley: